### PR TITLE
refactor: ignored iml files

### PR DESCRIPTION
**Pull request by pa1pal**
@sdumitriu I made changes of ignoring .iml files.

### CODE REVIEW

File: app/app.iml [hunk 1]
```diff
@@ -1,170 +0,0 @@
...
-    <orderEntry type="library" exported="" name="okhttp-3.6.0" level="project" />
-    <orderEntry type="library" exported="" scope="TEST" name="javawriter-2.1.1" level="project" />
-    <orderEntry type="library" exported="" scope="TEST" name="hamcrest-core-1.3" level="project" />
-    <orderEntry type="library" exported="" scope="TEST" name="exposed-instrumentation-api-publish-0.4.1" level="project" />
-    <orderEntry type="library" exported="" name="butterknife-8.0.1" level="project" />
-    <orderEntry type="library" exported="" name="logging-interceptor-3.6.0" level="project" />
-    <orderEntry type="library" exported="" name="butterknife-annotations-8.0.1" level="project" />
-    <orderEntry type="library" exported="" scope="TEST" name="junit-4.12" level="project" />
-  </component>
-</module>
```
Comment (sdumitriu) [inline app/app.iml:170]: The next two changes shouldn't be part of this commit.

Comment (pa1pal) [reply app/app.iml:170]: ok. I will send separate commits for this.

Comment (pa1pal) [reply app/app.iml:170]: done.
